### PR TITLE
Added support for the round floating buttons, typically used for big action calls

### PR DESCRIPTION
**Pull request by lgavillet**
http://materializecss.com/buttons.html#floating

screen shot 2016-07-05 at 11 29 09

### CODE REVIEW

**Comment by lgavillet** [comment]
Don't merge yet

**Comment by lgavillet** [comment]
Feel free

File: lib/mtl/rails/button_helper.rb [hunk 1]
```diff
@@ -22,6 +23,11 @@ def mtl_button(text, url, options = {})
         options[:class] = ['btn', Mtl.effects, options[:class]].compact
         link_to text, url, options
       end
+
+      def mtl_button_floating(text, url, options = {})
```
Comment (lwe) [inline lib/mtl/rails/button_helper.rb]: nitpick: ordering, please move above `mtl_button`

File: lib/mtl/rails/button_helper.rb [hunk 2]
```diff
@@ -10,6 +10,7 @@ module Rails
     # ```erb
     # <%= mtl_button "New", new_user_path # create a raised button %>
     # <%= mtl_button_flat "Delete", user_path(@user), method: :delete # create a flat button %>
+    # <%= mtl_button_floating "+", user_path(@user), method: :delete # create a floating button %>
```
Comment (lwe) [inline lib/mtl/rails/button_helper.rb]: nitpick 2: provide a better suited example, e.g.

```erb
<%= mtl_button_floating "+", new_user_path # create a floating button %> 
```

**Comment by lwe** [comment]
...and update CHANGES.md, danke

**Comment by lgavillet** [comment]
Append to the existing list or add a new version?

**Comment by lwe** [comment]
Thanks

**Pull request merged**